pyat/at/lattice/elements.py

```diff
@@ -618,6 +645,11 @@ def __setattr__(self, key, value):
 
         super(ThinMultipole, self).__setattr__(key, value)
 
+    @property
+    def integrated_strength(self):
```

Comment:
there is an identical function to this one in line 718 that does the multiplication by length. 
may be this is only a template function that is overloaded within the single elements?